packages/opencensus-instrumentation-https/src/https.ts

```diff
@@ -59,11 +59,26 @@ export class HttpsPlugin extends HttpPlugin {
       this.getPatchHttpsOutgoingRequest()
     );
     if (semver.satisfies(this.version, '>=8.0.0')) {
-      shimmer.wrap(
-        this.moduleExports,
-        'get',
-        this.getPatchHttpsOutgoingRequest()
-      );
+      shimmer.wrap(this.moduleExports, 'get', () => {
```

Comment:
@mayurkale22 I could not figure out how to change the `http` module to put this code into a method there, and then I could have just referenced it from this `https` module...

But everything references a specific version(test and code) so I did not spend more time in trying to figure out how to do this. 
Hence the reason I ended up coping the same code from the `http` module and just dumping it here.